data/json/items/armor.json

```diff
@@ -6671,6 +6671,33 @@
             "max_volume": 8,
             "flags": ["SHEATH_SWORD"]
         }
+    },{
+        "type" : "ARMOR",
+        "id" : "baldric",
+        "name" : "baldric",
+        "weight" : 1625,
+        "color" : "brown",
+        "covers" : ["TORSO"],
+        "symbol" : "[",
+        "description" : "A leather scabbard, big enough for anything up to a longsword, or even a bit larger than that.  Designed to be worn at the waist, secured by a shoulder belt.  Activate to sheath/draw a sword.",
+        "price" : 7500,
+        "material" : ["leather", "wood"],
+        "volume" : 8,
+        "bashing" : 4,
+        "warmth" : 0,
+        "phase" : "solid",
+        "encumbrance" : 7,
+        "coverage" : 10,
+        "material_thickness" : 1,
+        "flags" : ["VARSIZE", "WAIST", "OVERSIZE"],
```

Comment:
Holsters no longer have VARSIZE.

Scabbards currently do have it, but most likely should lose it too. Stuff that has adjustable belts and straps should not require tailoring to fit.